__tests__/app.test.js

```diff
@@ -41,3 +41,40 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("GET:200 responds with an article object, which contains all the necessary properties", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then((response) => {
+        const article = response.body;
```

Comment:
.expectObjectContaining or .toMatchObject are good jest matchers to use in this instance🙂